### ISSUE TITLE
Upgrade WaveDrom to the latest version (2.6.8)

### ISSUE THREAD

**Issue by am21** [yuzutech/kroki#647]
Bonjour,

It looks like the latest versions of wavedrom support a `rotate:` keyword to display the field names vertically, but I cannot get Kroki to do that, using kroki.io or the yuzutech container.

Any idea how to get the latest? It seems wavedrom has a bit-field plugin: https://observablehq.com/@drom/bitfield-testbox?collection=@drom/bitfield

**Comment by Mogztter** [comment]
Bonjour 😉 

> It looks like the latest versions of wavedrom support a rotate: keyword to display the field names vertically, but I cannot get Kroki to do that, using kroki.io or the yuzutech container.

Indeed!
The reason is that, I forgot to add WaveDrom in the Dependabot configuration: https://github.com/yuzutech/kroki/blob/master/.github/dependabot.yml

As a result, the projet doesn't receive notification when a new version is published on https://www.npmjs.com.

> Any idea how to get the latest? 

We need to add the following definition in `dependabot.yml`

```yaml
- package-ecosystem: npm
  directory: "/wavedrom"
  schedule:
    interval: daily
    time: "04:00"
  open-pull-requests-limit: 10
  labels:
    - "📜 javascript"
    - "🔗 dependencies"
```

Once this is done, WaveDrom will be updated and the next Kroki release will include the latest version of WaveDrom.

> It seems wavedrom has a bit-field plugin: observablehq.com/@drom/bitfield-testbox?collection=@drom/bitfield

As far as I understand, bitfield is maintained by the same author but it's an independent project that does not rely on WaveDrom: https://github.com/wavedrom/bitfield
Please create a dedicated issue, if you would like to see bitfield added.

If you want to create byte field diagrams, you should check out: https://github.com/Deep-Symmetry/bytefield-svg

**Comment by am21** [comment]
👍

**Comment by Mogztter** [comment]
It seems that the `rotate` keyword is only used in the bitfield diagram library. I will update the title to clarify that this issue is about WaveDrom being outdated.

**Comment by Mogztter** [comment]
Fixed in  2b787d3
